packages/flutter/test/painting/decoration_test.dart

```diff
@@ -347,4 +347,171 @@ void main() {
     expect(Decoration.lerp(const FlutterLogoDecoration(), const BoxDecoration(), 0.75), isInstanceOf<BoxDecoration>()); // ignore: CONST_EVAL_THROWS_EXCEPTION
     expect(Decoration.lerp(const FlutterLogoDecoration(), const BoxDecoration(), 1.0), isInstanceOf<BoxDecoration>()); // ignore: CONST_EVAL_THROWS_EXCEPTION
   });
+
+  test('paintImage BoxFit.none scale test', () {
+    for (double scale = 1.0; scale <= 4.0; scale += 1.0) {
+      final TestCanvas canvas = new TestCanvas(<Invocation>[]);
+
+      final Rect outputRect = Rect.fromLTWH(30.0, 30.0, 250.0, 250.0);
```

Comment:
nit: Please use `new` for a few more days while we await having a lint to remove all `new`s at once. (We want to be consistent in the meantime.)